enrollment-server/src/main/java/com/wultra/app/enrollmentserver/configuration/PowerAuthRestfulIntegrationConfiguration.java

```diff
@@ -1,45 +0,0 @@
-/*
- * PowerAuth Server and related software components
- * Copyright (C) 2023 Wultra s.r.o.
- *
- * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU Affero General Public License as published
- * by the Free Software Foundation, either version 3 of the License, or
- * (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU Affero General Public License for more details.
- *
- * You should have received a copy of the GNU Affero General Public License
- * along with this program.  If not, see <http://www.gnu.org/licenses/>.
- */
-package com.wultra.app.enrollmentserver.configuration;
-
-import io.getlime.security.powerauth.rest.api.spring.provider.MinimalClaimsUserInfoProvider;
-import io.getlime.security.powerauth.rest.api.spring.provider.UserInfoProvider;
-import lombok.extern.slf4j.Slf4j;
-import org.springframework.context.annotation.Bean;
-import org.springframework.context.annotation.Configuration;
-
-/**
- * Configuration of PowerAuth Restful Integration.
- *
- * @author Lubos Racansky, lubos.racansky@wultra.com
- */
-@Configuration
-@Slf4j
-public class PowerAuthRestfulIntegrationConfiguration {
-
-    /**
-     * Prepare user-info provider.
-     *
-     * @return user-info provider.
-     */
-    @Bean
-    public UserInfoProvider userInfoProvider() {
-        logger.info("Registering MinimalClaimsUserInfoProvider");
-        return new MinimalClaimsUserInfoProvider();
```

Comment:
For dev environment, the `MinimalClaimsUserInfoProvider` is not configured automatically anymore.